### PR TITLE
feat: visual testing

### PR DESCRIPTION
**Pull request by CuriouzK0d3r**
### Description of PR

Cypress Visual Testing added. mswjs mock data updated
What component(s) does this PR affect?

  - [ ] Back-End (Node.js, Next.js)
  - [X] Front-End (React.js)
  - [ ] Theming (CSS, React theme)
  - [X] Build System
  - [ ] Docs
  
  
  
  Does the PR require changes on other components? If yes, please mark the components:
  
  - [ ] Back-End (Node.js, Next.js)
  - [ ] Front-End (React.js)
  - [ ] Theming (CSS, React theme)
  - [ ] Build System
  - [ ] Docs
  
  
  
### Related Issue
  
  
  
  
  
  
  Resolves #
  
### Type
  
  - [ ] Bug fix (non-breaking change which fixes an issue)
  - [ ] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to change)
  - [ ] Docs update
  - [ ] None of the above
  
  
  
### Checklist:
  
  - [ ] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
  - [ ] This change requires a change in the documentation.
  - [ ] I have updated the documentation accordingly.
  >)`>>>)>

### CODE REVIEW

File: apps/artemis-web/utils/mock-sw/handlers.ts [hunk 1]
```diff
@@ -4,61 +4,34 @@ export const handlers = [
...
-            name: 'database',
-            running: true,
-            loading: false,
-            timestamp: '2019-12-04T13:44:19.725199',
-            __typename: 'view_processes',
-          },
-        ],
-      })
+        "view_processes":[
+          {"name":"autostarter-1","running":true,"loading":false,"timestamp":"2021-07-19T08:54:19.429356","__typename":"view_processes"},
```
Comment (vkotronis) [inline apps/artemis-web/utils/mock-sw/handlers.ts:8]: why do we append replica ids here while before we did not?

**Review by vkotronis — COMMENTED** [review]
The code looks OK with me, however I will need to setup and check a local instance to make sure that the live subscription queries work as expected. Will do that within the week.

File: apps/artemis-web/utils/mock-sw/handlers.ts (hunk 1 above)
Comment (vkotronis) [reply apps/artemis-web/utils/mock-sw/handlers.ts:8]: Yes just asking out of curiosity. Resolving.

**Comment by vkotronis** [comment]
TODO: @CuriouzK0d3r @vkotronis check locally with a real instance the GQL live behavior with the new state check changes (with testing disabled, so considering a prod environment). If all works OK, we can approve and merge.

File: apps/artemis-web/utils/hooks/use-graphql.d.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ type optionsType = {
   isMutation?: boolean;
   running?: boolean;
   name?: string;
+  isTesting?: boolean;
```
Comment (vkotronis) [inline apps/artemis-web/utils/hooks/use-graphql.d.ts:27]: btw where is this actually defined? Is it inherited from an env var?

**Review by vkotronis — APPROVED** [review]
tested, approving, just check small comment about the origin of the `isTesting` prop before merging (I tested in prod)

File: apps/artemis-web/utils/hooks/use-graphql.d.ts (hunk 2 above)
Comment (CuriouzK0d3r) [reply apps/artemis-web/utils/hooks/use-graphql.d.ts:27]: YES! the default is false, I am using this flag for CYPRESS testing

**Pull request merged**